theme/src/github.js

```diff
@@ -0,0 +1,74 @@
+export default {
```

Comment:
`prism-react-rendered` changed the `github` theme on v1.2.0 and it does not look right:

| Before | After |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/11280312/108733509-2cf49200-74f4-11eb-87b9-ebd35ffa1922.png) | ![image](https://user-images.githubusercontent.com/11280312/108733540-34b43680-74f4-11eb-854e-22278c5e09d2.png) |

So I decided to extract the previous version of the theme into this project to keep the docs consistent